Doc/library/math.rst

```diff
@@ -53,6 +53,8 @@ noted otherwise, all return values are floats.
 :func:`frexp(x) <frexp>`                              Mantissa and exponent of *x*
 :func:`isclose(a, b, rel_tol, abs_tol) <isclose>`     Check if the values *a* and *b* are close to each other
 :func:`isfinite(x) <isfinite>`                        Check if *x* is neither an infinity nor a NaN
+:func:`isnormal(x) <isnormal>`                        Check if *x* is a normal floating-point number
+:func:`issubnormal(x) <issubnormal>`                  Check if *x* is a subnormal floating-point number
```

Comment:
```suggestion
:func:`isnormal(x) <isnormal>`                        Check if *x* is a normal number
:func:`issubnormal(x) <issubnormal>`                  Check if *x* is a subnormal number
```

Reply:
It makes sense to use "floating-point" explicitly, as the result depends on the floating-point format. But then we should use it for `isfinite()` and `isinf()` too. I think that for consistency with existing documentation it is better to remove it.